plugins/inappauthplugin/plugin.cpp

```diff
@@ -100,7 +99,8 @@ Daemon::Plugins::InAppPlugin::beginUserInputInteraction(
                                   Q_ARG(Sailfish::Secrets::Result, Sailfish::Secrets::Result(Sailfish::Secrets::Result::Succeeded)),
                                   Q_ARG(QByteArray, QByteArray()));
         return Result(Result::Pending);
-    } else if (interactionParameters.promptText().startsWith(QLatin1String("Enter the new lock code for the plugin"))) {
+    } else if (interactionParameters.promptText().endsWith(
+                   QLatin1String("Enter the new passphrase for the plugin."))) {
```

Comment:
Entering new passphrase flows here only require typing password once. There is quite a high likelihood that user mistypes the password so repeat step should be required.

Reply:
Hrm, looking at https://github.com/sailfishos/sailfish-secrets/pull/15#issuecomment-365584951 I'm not sure how to set this, actually.  Will it automatically do it if I provide the repeatNewPasswordText and passwordMismatchText properties?

Reply:
@adenexter which properties do I need to set to make the prompt require the user to re-enter the new password?

Reply:
Also if user mistypes can we display error and ask user to try again?